javascript/javascript3/week3/shoppingCart.js

```diff
@@ -0,0 +1,142 @@
+// Shopping cart using Classes
+
+class Product {
+    constructor(name, price) {
+        this.name = name
+        this.price = price
+    }
+
+    convertToCurrency(currencyUnit) {
+        if (currencyUnit === 'euro') {
+            return this.price * 0.13
+        }
+        else if (currencyUnit === 'dollar') {
+            return this.price * 0.16
+        }
+        else if (currencyUnit === 'sk') {
+            return this.price * 1.38
+        }
+    }
+}
+
+class ShoppingCart {
+    constructor() {
+        this.products = []
+    }
+
+    addProduct(product) {
+        this.products.push(product)
+    }
+
+    removeProduct(product) {
+        const removedProductArray = this.products.filter((pro) => product.toLowerCase().trim() != pro.name.toLowerCase())
+        this.products = removedProductArray
+    }
+
+    searchProduct(productName) {
+        let result;
+        this.products.forEach((product) => {
+            if (product.name.toLowerCase() === productName.toLowerCase().trim()) {
+                result = true
+            }
+        })
+        return result
+    }
```

Comment:
This function works fine but you can also use the js `includes()` method :)

Reply:
Yes thank you I need to make more practise for be a more practical 😄
I will update that 👍